cmd/juju/commands/debuglog.go

```diff
@@ -70,19 +70,19 @@ append filtered messages:
 Include only unit mysql/0 messages; show a maximum of 50 lines; and then
 exit:
 
-    juju debug-log -T --include unit-mysql-0 --lines 50
+    juju debug-log --include unit-mysql-0 --limit 50
```

Comment:
so this one will exit once it shows a max 50 lines and not tail farther? I was expecting this would also need a --no-tail to meet the expectations of the sentence before it.

Reply:
Right, limit, limits the log and cuts off, it's essentially the same as:
`juju debug-log --include unit-mysql-0 --lines 50 --no-tail`

Reply:
We could change the message to not include limit at all or both. 
I added limit so that we have an example for each usage of the most options